src/Bicep.LangServer/Completions/BicepCompletionProvider.cs

```diff
@@ -373,6 +403,17 @@ private bool TryGetFilesForPathCompletions(Uri baseUri, string entered, [NotNull
             return true;
         }
 
+        private IEnumerable<CompletionItem> CreateFileCompletionItems(ParamsSemanticModel model, ParamsCompletionContext context, string entered, IEnumerable<Uri> fileUris, Uri cwdUri, Predicate<Uri> predicate, CompletionPriority priority) => fileUris
```

Comment:
Rather than duplicating this and the `CreateDirectoryCompletionItems` method, let's just refactor to pass `replacementRange` instead of `context`.